openvdb/openvdb/tools/VolumeToMesh.h

```diff
@@ -948,29 +971,28 @@ mergeVoxels(LeafType& leaf, const Coord& start, int dim, int regionId)
 // thinking that it is a constructor.
 template <class LeafType>
 inline bool
-isMergable(LeafType& leaf, const Coord& start, int dim,
+isMergable(const LeafType& leaf,
```

Comment:
Might as well fix the typo here - isMergeable

Reply:
Done